<launch>
    <!--
        IMPORTANT: Configure your nodes for evaluation within this launch file.
    -->
    <arg name="robot_frame" default="base_footprint"/>
    <arg name="config_name" default="control.yaml"/>

    <node name="path_follower" pkg="aro_control" type="path_follower.py" output="screen">

        <rosparam subst_value="true">
            map_frame: map
            odom_frame: odom
            robot_frame: $(arg robot_frame)
        </rosparam>

        <!-- loading of config files -->
        <rosparam file="$(find aro_control)/config/$(arg config_name)" />
        
        <remap from="path" to="path"/>
        <remap from="cmd_vel" to="cmd_vel"/>
    </node>
    
</launch>
